src/block/columns/edit.js

```diff
@@ -146,6 +148,20 @@ const Edit = props => {
 	)
 }
 
+// Load the polyfill for columns block :has() selector for Firefox
+const userAgent = navigator?.userAgent
+if ( userAgent?.indexOf( 'Firefox' ) !== -1 ) {
```

Comment:
This can trigger as true `userAgent?.indexOf( 'Firefox' )` if `userAgent` is `undefined`
```suggestion
if ( userAgent && userAgent.indexOf( 'Firefox' ) !== -1 ) {
```